.github/workflows/nightly.yml

```diff
@@ -125,9 +125,9 @@ jobs:
           path: ${{ env.BUILD_DIR }}/darktable-*.exe
           retention-days: 2
 
-  macOS:
+  MacOS:
     if: github.repository == 'darktable-org/darktable' || github.event_name == 'workflow_dispatch'
-    name: Nightly darktable.${{ github.sha }}.macosx
+    name: Nightly darktable.${{ github.sha }}.MacOS
```

Comment:
It really is macOS (TM) and Windows (TM)

Reply:
Also, please think about swapping the order re the label length comment - currently the OS simply not visible on e.g. https://github.com/darktable-org/darktable/actions/runs/4218855752

Reply:
I'm also tempted to throw out the "darktable.${{ github.sha }}" altogether, I don't think it brings any new information/utility here?

Reply:
Again, agreed!
Do you want me to remove that from the win64 action as well?

Reply:
Sure, let's review it all together as one big Nightly action improvement 😉